scico/functional/_functional.py

```diff
@@ -97,7 +97,7 @@ def prox(
             lam: Proximal parameter :math:`\lambda`.
             kwargs: Additional arguments that may be used by derived
                 classes. These include ``x0``, an initial guess for the
-                minimizer.
+                minimizer in the defintion of :math:`\mathrm{prox}`.
```

Comment:
Typo: defintion

Reply:
Fixed and flyspell-prog-mode added to init.el